docs/conf.py

```diff
@@ -67,7 +67,7 @@
 todo_include_todos = False
 
 intersphinx_mapping = {
-    'python': ('http://docs.python.org/', None),
+    'python': ('http://docs.python.org/3/', None),
```

Comment:
Oh, I see, it means that when we ref a Python standard library thing it goes to the Python 3 docs. That's definitely a good change.